internal/auth0/client/resource.go

```diff
@@ -120,6 +120,14 @@ func NewResource() *schema.Resource {
 				Optional:    true,
 				Description: "URLs that Auth0 may redirect to after logout.",
 			},
+			"oidc_backchannel_logout_urls": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
```

Comment:
We could consider adding a validation here to ensure we only pass 1 URL and error early, considering the API only accepts one url at the moment.